### DISCUSSION TITLE
can I graph it instead of table

### DISCUSSION THREAD

**Discussion by Amitaybi** [grafana/grafana-csv-datasource#61] [Q&A] [answered]
If I have 2 column: date + data
is there a way I can graph it instead of a table?

**Comment by marcusolsson** [comment, marked as answer]
By default, all values are considered to be text. To graph your query results, you need to define a schema with at least a Time field and a Number field.

Something like this: 

<img width="387" alt="Screenshot 2021-04-28 at 09 34 22@2x" src="https://user-images.githubusercontent.com/8396880/116364856-f04f5a80-a804-11eb-8cfa-d4e562432e0d.png">

**Comment by Amitaybi** [comment]
Works. Thanks for the tip.

**Reply by maheswarGR** [reply]
hi @Amitaybi @marcusolsson .. I have loaded csv file ..but i couldn't  view graph even after specifying time and number filed ..
Please help me out
![Screenshot (450)](https://user-images.githubusercontent.com/91525476/146924911-60b28f92-a8bb-47d0-8cea-74a14e065af3.png)
![Screenshot (451)](https://user-images.githubusercontent.com/91525476/146924947-cffeb76d-459d-46e5-b749-e1528c15d443.png)
